### PR TITLE
Add drizzle example

### PR DESCRIPTION
**Pull request by shiyuhang0**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/serverless-driver-drizzle-example.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    `phone` varchar(256)
+   );
+   ```
+
+2. In the root directory of your project, create a file named `hello-word.ts` and add the following code:
+
+   ```ts
+   import { connect } from '@tidbcloud/serverless';
+   import { drizzle } from 'drizzle-orm/tidb-serverless';
+   import {mysqlTable, serial, text, varchar } from 'drizzle-orm/mysql-core';
```
Comment (zhangyangyu) [inline tidb-cloud/serverless-driver-drizzle-example.md:103]:
```suggestion
   import { mysqlTable, serial, text, varchar } from 'drizzle-orm/mysql-core';
```

File: tidb-cloud/serverless-driver-drizzle-example.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+   ```
+
+2. In the `app` directory of your project, create a file `/api/edge-function-example/route.ts` and add the following code:
+
+   ```ts
+   import { NextResponse } from 'next/server';
+   import type { NextRequest } from 'next/server';
+   import { connect } from '@tidbcloud/serverless';
+   import { drizzle } from 'drizzle-orm/tidb-serverless';
+   import {mysqlTable, serial, text, varchar } from 'drizzle-orm/mysql-core';
```
Comment (zhangyangyu) [inline tidb-cloud/serverless-driver-drizzle-example.md:202]:
```suggestion
   import { mysqlTable, serial, text, varchar } from 'drizzle-orm/mysql-core';
```

File: tidb-cloud/serverless-driver-drizzle-example.md [hunk 3]
```diff
@@ -69,15 +69,23 @@ To complete this tutorial, you need the following:
...
-    ```
+2. On the overview page, click **Connect** in the upper-right corner, select `Serverless Driver` in the **Connect With** drop-down box, and then click **Generate Password** to create a random password.
+
+    > **Tip:**
+    >
+    > If you have created a password before, you can either use the original password or click **Reset Password** to generate a new one.
+
+    The connection string looks like this:
+
+    ```shell
```
Comment (qiancai) [inline tidb-cloud/serverless-driver-drizzle-example.md:82]:
```suggestion
    ```
```

File: tidb-cloud/serverless-driver-drizzle-example.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+
+   ```sql
+   CREATE TABLE `test`.`users` (
+    `id` BIGINT PRIMARY KEY auto_increment,
+    `full_name` TEXT,
+    `phone` VARCHAR(256)
+   );
+   ```
+
+2. In the root directory of your project, create a file named `hello-word.ts` and add the following code:
```
Comment (hfxsd) [inline tidb-cloud/serverless-driver-drizzle-example.md:106]: It's `hello-world.ts` in L145.
```suggestion
2. In the root directory of your project, create a file named `hello-world.ts` and add the following code:
```

**Review by hfxsd — APPROVED** [review]

**Review by zhangyangyu — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**